ios/FluentUI/Core/Theme/Tokens/TokenizedControlView.swift

```diff
@@ -28,3 +28,6 @@ extension TokenizedControlView {
         return self
     }
 }
+
+/// Common base type alias for all `state` objects.
+typealias ControlState = NSObject & ObservableObject & Identifiable
```

Comment:
Didn't we want to change the name from state?

Reply:
I think that ship may have sailed at this point, but even if we eventually do I think I'd rather we focus on consistency first and foremost here.